### DISCUSSION TITLE
Virtual Keyboard

### DISCUSSION THREAD

**Discussion by CiberNin** [r0x0r/pywebview#850] [Q&A]
So I'm trying to make an raspberry pi kiosk. The screen is touch enabled with no provision for a separate keyboard, so I would like to have a virtual keyboard. 
Do you think the best way is to bring some sort of javascript library, use an entirely separate program like Florence, or is there some way to install an extension to webkit?
Just asking for an opinion.

**Comment by r0x0r** [comment]
There are no keyboard extension  for webkit or such.
I would go with a js library route myself
